ngraph_bridge/ngraph_data_cache.h

```diff
@@ -53,13 +53,14 @@ class NgraphDataCache {
   // it will create item, put it in the cache and returns item and status
   std::pair<Status, ValueType> LookUpOrCreate(
       KeyType key,
-      std::function<std::pair<Status, ValueType>()> callback_create_item,
-      std::function<void(ValueType)> callback_destroy_item);
+      std::function<std::pair<Status, ValueType>(KeyType)> callback_create_item,
+      std::function<void(ValueType)> callback_destroy_item, bool& cache_hit);
```

Comment:
note-to-self: `callback_create_item` is a function that now accepts a `KeyType` (instead of void)